src/XML/src/FMI3/fmi3_xml_model_structure.c

```diff
@@ -232,12 +219,14 @@ int fmi3_xml_parse_dependencies(fmi3_xml_parser_context_t *context,
             }
             if(!ch) break;
             if(sscanf(cur, "%d", &ind) != 1) {
+                // TODO: Rework this error; list actual type
                 fmi3_xml_parse_error(context, "XML element 'Unknown': could not parse item %d in the list for attribute 'dependencies'",
                     numDepInd);
                ms->isValidFlag = 0;
                return 0;
             }
             if(ind < 1) {
+                // TODO: Rework this error; list actual type
```

Comment:
Multiple occasions of this one, ties into the same issue as discussed in https://github.com/modelon-community/fmi-library/pull/44#discussion_r1185144221

Reply:
Not fixing this in this MR.